### PR TITLE
Update convention for images (issue 1261)

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/README.md [hunk 1]
```diff
@@ -236,6 +237,13 @@ Each guide must have an `images/` subdirectory with `images/common` symlink into
 Images local to the guide shall be kept in the `images/` directory.
 Images which are supposed to be reused across guides shall be kept in the `images/common` directory.
```
Comment (adamlazik1) [inline guides/README.md:238]:
```suggestion
Images which are supposed to be reused across guides shall be kept in the `images/common/` directory.
```

File: guides/README.md [hunk 2]
```diff
@@ -236,6 +237,13 @@ Each guide must have an `images/` subdirectory with `images/common` symlink into
...
 Subdirectories can be created and are actually recommended.
+
 To insert an image, use `image::common/global_image.png` or `image::local_image.png`.
 
+Upstream diagrams should be created using [diagrams.net](https://www.diagrams.net/).
+The editable diagram in `drawio` format shall be placed in `guides/image-sources`.
```
Comment (adamlazik1) [inline guides/README.md:244]:
```suggestion
You should create upstream diagrams using [diagrams.net](https://www.diagrams.net/).
Place the editable diagram in `drawio` format in `guides/image-sources/`.
```

**Review by maximiliankolb — APPROVED** [review]
thanks for adding this!

File: guides/README.md [hunk 3]
```diff
@@ -234,8 +235,15 @@ Snippets do not require an ID.
 
 Each guide must have an `images/` subdirectory with `images/common` symlink into the `common/images` directory.
```
Comment (adamlazik1) [inline guides/README.md:236]:
```suggestion
Each guide must have an `images/` subdirectory with `images/common` symlink into the `common/images/` directory.
```
Not including it in symlink on purpose.

**Review by adamlazik1 — APPROVED** [review]
Sorry to not notice that last bit earlier. Other than that, LGTM :+1:

**Comment by Lennonka** [comment]
Cherry-picked:
- bbffbefb9..df5c25077  3.4 -> 3.4
- 838e6c364..7841c7547  3.3 -> 3.3
- 090c3793d..e7c0b3a14  3.2 -> 3.2
- 4fee89ad7..7c6b277bf  3.1 -> 3.1

**Pull request merged**